README.md

```diff
@@ -10,7 +10,7 @@ Organizer is a Rust script to organize the Download directory in diferent direct
 1. Install rustup.
 2. Download this repository.
 ```console
-git clone https://github.com/ghaerdi/file-organizer.git
+git clone https://github.com/yacosta738/file-organizer.git

```

Comment:
Revert this

Reply:
the repo URL reverted 

![bitmoji](https://sdk.bitmoji.com/render/panel/12f3a5e1-c85f-49cd-94f4-6dce9cb18ef1-db1ffc10-9e7f-4bea-a043-e744f59d5043-v1.png?transparent=1&palette=1&width=246)